public/static/locales/fr/form-builder.json

```
@@ -203,6 +203,9 @@
       "title": "Plus d'informations"
     },
     "email": "Votre courriel",
+    "department": "Ministère ou organisme",
+    "branch": "Direction ou secteur",
+    "jobtitle": "Titre de poste",
```

Wondering if we could use camelcase for this. The issue with just using `title` is that there is already a field for that, which is the title of the page.

```suggestion
    "jobTitle": "Titre de poste",
```